melatonin/helpers/fps.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <juce_audio_processors/juce_audio_processors.h>
```

Comment:
Do we need decency on juce_audio_processors juce module here? I think you can just use juce_gui_basics, since VBlankAttachment is in it